addons/website_event/static/src/scss/event_templates_common.scss

```diff
@@ -193,3 +193,18 @@ $o-wevent-color: mix(map-get($theme-colors, 'primary'), $body-color, $o-wevent-c
         z-index: $zindex-dropdown - 1;
     }
 }
+
+.o_wevent_cta  {
```

Comment:
Why not using `.btn-primary` directly ? 🤔
Beside saving a couple of css lines, using the right class will ensure that the button is readable regardless by theme customizations

Reply:
As discussed, we need to create a new design for these components that are rendered inside the page but that are not part of the content.

Reply:
Yes, ok,  thanks